docs/Changelog.md

```diff
@@ -6728,7 +6728,7 @@ This version of the operator has been available since version 7 of the default O
 
 <dl>
 <dt><tt>Y</tt> (optional) : T</dt>
-<dd>A tensor that concats all the intermediate output values of the hidden. It has shape `[seq_length, num_directions, batch_size, hidden_size]`. </dd>
+<dd>A tensor that concats all the intermediate output values of the hidden. If a sequence with length less than `seq_length` is specified via sequence_lens, values past the end of that sequence will be set to zero. This tensor has shape `[seq_length, num_directions, batch_size, hidden_size]`.</dd>
```

Comment:
The end of the sequence shouldn't be set to zero.

Reply:
Even in C++, you have the length of the sequence, so the remaining part shouldn't be read.

Reply:
Ok. That was the main point of this PR, so I'll just close it.